### ISSUE TITLE
Wrong SWF in the maven repo?

### ISSUE THREAD

**Issue by nicoulaj** [flex-users/flex-iframe#52]
Originally filed by dirkvran...@gmail.com on 2009-11-09T10:56:55

I was trying to update my project to the newest flex-iframe version 1.4.1 and 
downloaded the SWF from the maven repo. However the file seems to be corrupt. 
When plugging it in to the app I get an error on compile time that the SWF 
cannot be read. When trying to open it with WinRar I also get a corrupt 
error. Downloading the zip file and getting the SWF from the bin directory 
works like a charm. So maybe the generated SWF on the maven repo is corrupt?

**Comment by nicoulaj** [comment]
Updated by Julien.N...@gmail.com on 2009-11-09T12:18:04

I checked and it looks okay to me... I use it with Maven too without any problem.
This is a SWC and not a SWF, maybe you made an error in the  element
inside  ?
You can also try to delete it from your local Maven repository to force downloading
it again.
Does it work with the previous versions or did you just setup the Maven dependency ?

**Comment by nicoulaj** [comment]
Updated by dirkvran...@gmail.com on 2009-11-09T13:09:49

What I do is I download the new SWC from your maven repo manually and add it to our own 
nexus repository. Did the exact same thing for the two previous versions and that 
worked perfect, but not for the last one. Anyway I copied the SWC from the zip and 
added that one to our nexus repo and that seems to work! Maybe something went wrong 
with my download as you use it yourself with success.
